encoders/firebase-decoders-json/src/json/java/com/google/firebase/decoders/json/JsonDataDecoderBuilderContext.java

```diff
@@ -274,6 +244,35 @@
     }
   }
 
+  @SuppressWarnings("unchecked")
+  private <T> T defaultValue(TypeToken<T> typeToken) {
+    if (typeToken instanceof TypeToken.ArrayToken) {
+      return convertGenericListToArray(null, (TypeToken.ArrayToken<T>) typeToken);
```

Comment:
Does it have to be `null` here, could we use `Collections.emptyList()` instead?

Reply:
Both of them works, but `Collections.emptyList()` is better, since it explicitly indicates the default value of array is `[]`. Great Call!